rootfs/scheduler/resources/pod.py

```diff
@@ -120,6 +120,11 @@ def manifest(self, namespace, name, image, **kwargs):
             'spec': {}
         }
 
+        # pod annotations
+        annotations = self._get_json_from_app('pod_annotations', **kwargs)
+        if annotations:
+            manifest['metadata']['annotations'] = annotations
```

Comment:
we need to ensure we're not over-writting any of deis annotations, are there any tests for it?

Reply:
Deis itself does not add any annotations to the `pod` template metadata, it adds the `kubernetes.io/change-cause` and `deployment.kubernetes.io/revision` annotations to the `deployment` metadata only, so i don't think there is the need to test for any specific annotations.

Reply:
👍